src/services/account/hooks/index.js

```diff
@@ -281,8 +281,9 @@ const restrictToUsersSchool = async (context) => {
 };
 
 const validateUserName = async (context) => {
-	const accountService = context.app.service('accounts');
-	const { systemId } = context.method === 'create' ? context.data : await accountService.get(context.id);
+	const accountService = context.app.service('nest-account-service');
+
+	const { systemId } = context.method === 'create' ? context.data : await accountService.findById(context.id);
```

Comment:
This was difficult to read before the refactoring and it still is. So it is not worse.
But I prefer an if statement here.

Reply:
Checked OK. Spread into if-else block